CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 Change Log
 ===
 
+v3.0.0-dev.1
```

Comment:
how about we use "alpha" now that we're no longer tied to the TS "dev" terminoogy?